cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,21 @@
+class ShippingstepPage{
+
+    private checkBox: string;
+    private carrier: string;
+
+    constructor(){
+        this.checkBox = "[name=cgv]";
+        this.carrier = "[name=processCarrier]";
+    }
+
+    public checkingBox(): void {
```

Comment:
name of methods should use a verb in the infinitive, also, the name could be more descriptive, something like acceptTerms or something that indicates that we are accepting the terms and conditions.